src/test/java/net/sf/log4jdbc/PreparedStatementSpyTest.java

```diff
@@ -156,18 +159,24 @@ void invalidShowSqlParamsValue() throws Throwable {
         withFreshlyLoaded(StatementSpy.class).exec(InvalidShowSqlParamsValueCase.class);
     }
 
-    interface Case {
+  interface Case {
+    ConnectionSpy cs = mock(ConnectionSpy.class);
```

Comment:
While factoring out common dependencies definitely makes test cases cleaner, the idea of the `Case` was to be independent of any case specifics and to have a single responsibility -- be a carrier for a code that has to be executed in a context of a custom class loader. If later there will be more cases not depending on `ConnectionSpy` and `PreparedStatement` mocks, more mocks will be added here making test setup harder to read because it's similar to moving state to a global scope where it gets mixed with other states making this mix harder to read and maintain.

I don't think though there will be many more tests and definitely it won't hurt here, so it's kinda thinking aloud, no need to do something. Cases look cleaner now.